python/arb.cpp

```diff
@@ -0,0 +1,235 @@
+#include <vector>
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include <domain_decomposition.hpp>
+#include <model.hpp>
+#include <load_balance.hpp>
+#include <profiling/meter_manager.hpp>
+#include <recipe.hpp>
+#include <rss_cell.hpp>
+
+#include "print.hpp"
+#include "recipe.hpp"
+
+namespace pb = pybind11;
+
+template <typename T>
+arb::util::any wrap_any(T value) {
+    return arb::util::any(std::move(value));
+}
```

Comment:
I don't think this is being used…

Reply:
Right you are.
It is left over cruft from an earlier version.